web/admin_service.go

```diff
@@ -0,0 +1,161 @@
+package web
+
+import (
+	"context"
+	"errors"
+	"strconv"
+
+	"github.com/autograde/quickfeed/admin"
+	pb "github.com/autograde/quickfeed/ag"
+	"github.com/autograde/quickfeed/database"
+	"github.com/autograde/quickfeed/scm"
+	scms "github.com/autograde/quickfeed/scm"
+	"github.com/autograde/quickfeed/web/config"
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+)
+
+type AdminService struct {
+	logger *zap.SugaredLogger
+	db     database.Database
+	app    *scms.GithubApp
```

Comment:
Should this be an `scm.Admin` interface that only expose the SCM methods that the `AdminService` needs??